aries-site/src/pages/index.js

```diff
@@ -29,7 +30,7 @@ const Index = () => {
     }
   }, []);
 
-  const data = Object.values(homeData);
+  const data = structure.filter(page => page.pages);
```

Comment:
`data` is a bit generic here, wonder if `topicList` is more explicit and would communicate to developer what this data represents?

Reply:
Agreed, we can be more specific.